packaging/macos/sunshine.rb

```diff
@@ -0,0 +1,48 @@
+require "language/node"
+
+class @PROJECT_NAME@ < Formula
+  desc "@PROJECT_DESCRIPTION@"
+  homepage "@PROJECT_HOMEPAGE_URL@"
+  url "@GITHUB_CLONE_URL@",
+    tag: "@GITHUB_BRANCH@"
+  version "@PROJECT_VERSION@"
+  license all_of: ["GPL-3.0-only"]
+  head "@GITHUB_CLONE_URL@"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "curl"
+  depends_on "miniupnpc"
+  depends_on "node"
+  depends_on "openssl"
+  depends_on "opus"
```

Comment:
Should we add avahi?

Reply:
when I try `brew install avahi`
avahi: Linux is required for this software.